tests/unit/CMakeLists.txt

```diff
@@ -1,4 +1,9 @@
 bout_update_submodules()
+
+# disable gtest pthreads for proper execution of Death tests (on some platforms)
+add_definitions(-DGTEST_HAS_PTHREAD=0)
```

Comment:
This adds the macro to all targets, we should limit it to just the ones that need it, presumably `serial_tests`?